metadata-core/src/main/java/com/hashmap/haf/metadata/core/data/resource/jdbc/model/JdbcResource.java

```diff
@@ -1,12 +1,21 @@
 package com.hashmap.haf.metadata.core.data.resource.jdbc.model;
 
 import com.hashmap.haf.metadata.core.data.resource.DataResource;
+import com.mysql.jdbc.CommunicationsException;
+import com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException;
+import lombok.extern.slf4j.Slf4j;
 
+import java.sql.*;
+import java.util.HashMap;
+import java.util.Map;
+
+@Slf4j
 public class JdbcResource extends DataResource<JdbcResourceId> {
 
     private String dbUrl;
     private String username;
     private String password;
+    final String JDBC_DRIVER = "com.mysql.jdbc.Driver";
```

Comment:
Just validate, how will you handle different JDBC drivers e.g., with postgres...do you need to rename Resource to MySqlResource?